source/creating_doc/quickstart_rst.rst

```diff
@@ -0,0 +1,139 @@
+Quick Start: Writing RST
+########################
+
+RST is very powerful, and flexible.  Below we provide a quick guide for how you
+can use it to write Open edX documentation.
+
+For more details check out the `RST Primer`_
+
+.. _RST Primer: https://docutils.sourceforge.io/docs/user/rst/quickstart.html
+
+Headings
+********
+
+.. code-block:: RST
+
+   Heading 1
+   #########
```

Comment:
I've been doing the exact opposite:

```
h1
--

h2
==

h3
**

h4
##
```

But I'll admit that your way looks much better visually. Just a heads up that you might be fighting an uphill battle with this one. Anway, I'll support whichever one you choose.

Reply:
Yea, I really struggled between the two directions for this but ultimately landed where I did because the raw text visually tracks with what I would expect in terms of line density.  I'm gonna leave it this way for now but I definitely struggled with the choice.